src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerHelpers/InteropHelpers.cs

```diff
@@ -391,23 +390,39 @@ internal static unsafe void FixupMethodCell(IntPtr hModule, MethodFixupCell* pCe
         }
 
 #if TARGET_WINDOWS
-        private static unsafe IntPtr GetProcAddressWithSuffix(IntPtr hModule, byte* methodName, byte suffix)
+        private static unsafe IntPtr GetProcAddressWithMangling(IntPtr hModule, byte* methodName, MethodFixupCell* pCell)
+        {
+            IntPtr pMethod = Interop.Kernel32.GetProcAddress(hModule, methodName);
+#if TARGET_X86
+            if (pMethod == IntPtr.Zero && pCell->IsStdcall)
+            {
+                int nameLength = string.strlen(methodName);
+                // We need to add an extra bytes for the prefix, null terminator and stack size suffix ('@' and up to 10 digits)
+                byte* probedMethodName = stackalloc byte[nameLength + 13];
```

Comment:
I think it would be easier to call out each component in a `const` local. I say that because '@' is a single character but also 2 bytes and we also have a `null`. Based on the comment I struggling to get to 13.

'@' 1 character or 2 bytes - we are using bytes so I assume 2?
'null' 1 byte or 2 bytes - 1 byte for for UTF-8 or 2 bytes for UTF-16, I assume 1 in this based on 13?
'10 digits' - so this is in UTF-8, right? So does that means '@' should be 1 byte?

- 1 for '@' and 1 for 'null' plus 10 is 12 bytes
- 2 for '@' and 2 for 'null' plus 10 characters should be 24 bytes
- 2 for '@' and 1 for 'null' plus 10 UTF-8 characters is 13 bytes, but mixes sizes

Reply:
1 for `_` prefix + name length + 1 for `@` + 10 for digits + 1 for null (all in bytes).

Reply:
I updated the comment to make it a bit clearer.

Reply:
Ah. Missed the prefix (`_`). Thank you!